docs/module_docs/group_isomers/annotate_possible_isomers.md

```diff
@@ -0,0 +1,43 @@
+# Annotate possible isomers
```

Comment:
I would not call it isomers because the module searches for isomers and isobars.